test/create.ts

```diff
@@ -27,7 +32,7 @@ test('supports instance defaults', withServer, async (t, server, got) => {
 			'user-agent': 'custom-ua-string'
 		}
 	});
-	const headers = await instance('').json();
+	const headers = await instance('').json() as Headers;
```

Comment:
```suggestion
	const headers = await instance('').json<Headers>();
```

Reply:
This couldn't be done because TS complains `Headers` is not assignable to `JsonValue`. I am still thinking of a way to solve that ...

I tried both ways below with no luck:
```js
const headers = await instance('').json<Headers>();
```

```js
const headers: Headers = await instance('').json();
```

Reply:
Like I've commented elsewhere, don't use `JsonValue`. It creates more problems than it solves.

Generally, don't use `as` unless absolutely needed, as it can hide problem, since it just force casts anything to anything.